src/chronify/time_series_mapper_base.py

```diff
@@ -32,13 +31,22 @@ def __init__(
         metadata: MetaData,
         from_schema: TableSchema,
         to_schema: TableSchema,
+        data_adjustment: Optional[TimeBasedDataAdjustment] = None,
+        wrap_time_allowed: bool = False,
     ) -> None:
         self._engine = engine
         self._metadata = metadata
         self._from_schema = from_schema
         self._to_schema = to_schema
-        # self._from_time_config = from_schema.time_config
-        # self._to_time_config = to_schema.time_config
+        self._from_time_config = from_schema.time_config
```

Comment:
For each of the mapper classes, I am getting a ton of mypy [union-attr] errors. Is there a way I can restructure the base class to make them go away?

Reply:
I pushed a commit that fixes all mypy errors...hope that's ok, it's easier than to try to explain over comments. I also disabled Spark tests, which were also failing in Zack's PR.